### ISSUE TITLE
Remove min-width from CSS

### ISSUE THREAD

**Issue by beruic** [yourlabs/django-autocomplete-light#830]
In [django-autocomplete-light/src/dal_select2/static/autocomplete_light/select2.css on line 2](https://github.com/yourlabs/django-autocomplete-light/blob/master/src/dal_select2/static/autocomplete_light/select2.css#L2) mimn-width is set to `20em`. This has caused me some trouble, and ultimately I has to override it from another CSS file.

The issue is that `20em` is too wide some places, because of changing fonts and font sizes. I had to override it to 
```
.select2-container {
    min-width: 0 !important;
}
```
This file smells a bit like it is only fixing Django specific issues in the admin. If that is the case, it should be possible to avoid it in your own views.

**Comment by jpic** [comment]
Absolutely, an alternative would be to prepend the selector in our select2.css to make it match only admin templates, would that work ?

**Comment by beruic** [comment]
Yes it would :)

**Comment by jpic** [comment]
Awesome, you got it :guitar:

**Comment by jaredaus** [comment]
Lifesaver! this was causing me an issue when displaying autocomplete options on mobile devices (in bootstrap)!

**Comment by jpic** [comment]
@jaredaus contribution welcome :)
